designs/simon/src/collar.mjs

```diff
@@ -24,7 +25,7 @@ function simonCollar({
 }) {
   const draft = function (tweak = 1) {
     const length = measurements.neck * (1 + options.collarEase - options.collarGap) * tweak
-    const width = store.get('collarStandWidth') * 1.2 * (1 + options.collarRoll)
+    const width = store.get('collarStandWidth') * options.collarWidth * (1 + options.collarRoll)
```

Comment:
Looks like the "collarRoll" setting already does basically the same as the new option. It just has a super small range and applies on top of the 1.2 factor. Not sure if anything needs to be done with that redundancy.

Reply:
Good catch! I'd mistaken `collarRoll` for something else. Based on the description, I thought it was used for the line for the undecollar, to make the collar "roll" back and hide the seam between the two near-identical collar pieces. I don't think having both options is necessary. Maybe this PR should be replace with clarifying + expanding the doc for `collarRoll`, and expanding it's range?

Though getting rid of the fixed 1.2 multiplier in the code and changing the default for `collarRoll` to compensate would likely be a breaking change that those who used a custom `collarRoll` would have to compensate for w/old patterns. I'm not sure what the best way to handle such a situation is.

Reply:
I made a commit that removes the redundancy, by having `collarRoll` affect the undercollar line. It seems to me this was the intended use, *or* there's an undercollar <-> collar stand mixup *somewhere* that is making `collarRoll` a very confusing option to me. Once joost reviews the PR, I'll let him decide if this is indeed a fix, or if there's another way this situation should be handled :D

Reply:
I don’t think the collar roll option pertains to the issue I filed #6907 about. Collar roll seems to just affect the difference between the collar and undercollar. I would like to be able to adjust the height of the collar/undercollar and collar stand separately, instead of them always changing together like they are now.